languages/en.ini

```diff
@@ -14,6 +14,7 @@ account_normal_checkouts = "Items due later"
 account_requests_available = "Available for Pickup"
 account_requests_in_transit = "In Transit"
 Add a Library Card = "Add a Library Card"
+Add a Library Card using Shibboleth login = "Add a Library Card using Shibboleth login"
```

Comment:
This just occurred to me: I think it might make sense to change the default wording to align with the actual login option "Institutional Login". I believe that the term "Shibboleth" is something that a user would not typically be aware of.